cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -135,7 +145,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(TestBase.class);
 
   @ClassRule
-  public static TemporaryFolder tmpFolder = new TemporaryFolder();
+  public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
```

Comment:
This change is not backward compatible (public field) and TestBase is a public class.